app/components/Groups.spec.tsx

```diff
@@ -0,0 +1,22 @@
+import { render, screen } from '@testing-library/react'
```

Comment:
most examples in https://github.com/vitest-dev/vitest/tree/main/examples use a dedicated `test` folder, I'd prefer that pattern over mixing code and tests

Reply:
Okay, but what's the indentend file structure we want to follow? 

I have seen this in the past: 
```
[app]
   [routes]
        index.tsx
        
[test]
   [app]
      [routes]
          index.spec.tsx
```

At the moment I use this, which is the reason I thought the test directory is redundant:
```
[app]
   [routes]
        [home]
            index.tsx
            index.spec.tsx
```

Reply:
test-app-routes or test-routes should work